examples/python-cicd-with-cloudbuilder/python-cloud-builder/Dockerfile

```diff
@@ -1,3 +1,7 @@
+# This Dockerfile is used to build a custom cloud builder, shown in step 3 of README.md
+# It pulls the public python 3 image and then installs virtualenv so that a virtual environment can
```

Comment:
Should Python be capitalized here? I'm actually not sure.

Reply:
I'm honestly not sure.  Python the language is uppercase, the container image is python.  Beats me.  I'll capitalize it.